edu-services/gradebook-service/impl/src/java/org/sakaiproject/component/gradebook/GradebookServiceHibernateImpl.java

```diff
@@ -3223,11 +3222,21 @@ public Object doInHibernate(final Session session) throws HibernateException {
 				// set date recorded
 				cg.setDateRecorded(gr.getDateRecorded());
 
+				// set entered points
+				cg.setEnteredPoints(gr.getEnteredPoints());
```

Comment:
This is still setting the record in the database?

Reply:
This is a new column on DB. I'll do another PR on the scripts.